### PR TITLE
Fixed the issue happening while conpiling the upstream nvme-cli

### PR DESCRIPTION
**Pull request by Naresh-ibm**
compilation of upstream nvme-cli was failing. this script will fix this problem and it work fine

### CODE REVIEW

**Comment by Naresh-ibm** [comment]
`[root@sys ssd]# avocado run --max-parallel-tasks=1 old-nvmetest.py -m nvmetest.py.data/nvmetest.yaml 
Fetching asset from old-nvmetest.py:NVMeTest.test_firmware_upgrade
Fetching asset from old-nvmetest.py:NVMeTest.test_create_max_ns
Fetching asset from old-nvmetest.py:NVMeTest.test_create_full_capacity_ns
Fetching asset from old-nvmetest.py:NVMeTest.testformatnamespace
Fetching asset from old-nvmetest.py:NVMeTest.testread
Fetching asset from old-nvmetest.py:NVMeTest.testwrite
Fetching asset from old-nvmetest.py:NVMeTest.testcompare
Fetching asset from old-nvmetest.py:NVMeTest.testflush
Fetching asset from old-nvmetest.py:NVMeTest.testwritezeroes
Fetching asset from old-nvmetest.py:NVMeTest.testwriteuncorrectable
Fetching asset from old-nvmetest.py:NVMeTest.testdsm
Fetching asset from old-nvmetest.py:NVMeTest.testreset
Fetching asset from old-nvmetest.py:NVMeTest.testreset_sysfs
Fetching asset from old-nvmetest.py:NVMeTest.testsubsystemreset
Fetching asset from old-nvmetest.py:NVMeTest.test_create_namespaces
Fetching asset from old-nvmetest.py:NVMeTest.test_delete_all_ns
JOB ID     : c8bfdfd68c7ce5164bc809a136a1f73945b3aa88
JOB LOG    : /home/OpTest/avocado-fvt-wrapper/results/job-2023-09-22T00.38-c8bfdfd/job.log
 (01/32) old-nvmetest.py:NVMeTest.test_firmware_upgrade;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: STARTED
 (01/32) old-nvmetest.py:NVMeTest.test_firmware_upgrade;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: CANCEL: firmware url not given (12.79 s)
 (02/32) old-nvmetest.py:NVMeTest.test_firmware_upgrade;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: STARTED
 (02/32) old-nvmetest.py:NVMeTest.test_firmware_upgrade;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: CANCEL: firmware url not given (0.35 s)
 (03/32) old-nvmetest.py:NVMeTest.test_create_max_ns;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: STARTED
 (03/32) old-nvmetest.py:NVMeTest.test_create_max_ns;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: PASS (6.24 s)
 (04/32) old-nvmetest.py:NVMeTest.test_create_max_ns;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: STARTED
 (04/32) old-nvmetest.py:NVMeTest.test_create_max_ns;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: PASS (0.84 s)
 (05/32) old-nvmetest.py:NVMeTest.test_create_full_capacity_ns;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: STARTED
 (05/32) old-nvmetest.py:NVMeTest.test_create_full_capacity_ns;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: PASS (6.12 s)
 (06/32) old-nvmetest.py:NVMeTest.test_create_full_capacity_ns;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: STARTED
 (06/32) old-nvmetest.py:NVMeTest.test_create_full_capacity_ns;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: PASS (0.95 s)
 (07/32) old-nvmetest.py:NVMeTest.testformatnamespace;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: STARTED
 (07/32) old-nvmetest.py:NVMeTest.testformatnamespace;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: PASS (16.19 s)
 (08/32) old-nvmetest.py:NVMeTest.testformatnamespace;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: STARTED
 (08/32) old-nvmetest.py:NVMeTest.testformatnamespace;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: PASS (10.91 s)
 (09/32) old-nvmetest.py:NVMeTest.testread;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: STARTED
 (09/32) old-nvmetest.py:NVMeTest.testread;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: PASS (5.71 s)
 (10/32) old-nvmetest.py:NVMeTest.testread;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: STARTED
 (10/32) old-nvmetest.py:NVMeTest.testread;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: PASS (0.37 s)
 (11/32) old-nvmetest.py:NVMeTest.testwrite;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: STARTED
 (11/32) old-nvmetest.py:NVMeTest.testwrite;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: PASS (5.64 s)
 (12/32) old-nvmetest.py:NVMeTest.testwrite;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: STARTED
 (12/32) old-nvmetest.py:NVMeTest.testwrite;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: PASS (0.39 s)
 (13/32) old-nvmetest.py:NVMeTest.testcompare;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: STARTED
 (13/32) old-nvmetest.py:NVMeTest.testcompare;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: PASS (5.73 s)
 (14/32) old-nvmetest.py:NVMeTest.testcompare;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: STARTED
 (14/32) old-nvmetest.py:NVMeTest.testcompare;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: PASS (0.37 s)
 (15/32) old-nvmetest.py:NVMeTest.testflush;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: STARTED
 (15/32) old-nvmetest.py:NVMeTest.testflush;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: PASS (5.65 s)
 (16/32) old-nvmetest.py:NVMeTest.testflush;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: STARTED
 (16/32) old-nvmetest.py:NVMeTest.testflush;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: PASS (0.38 s)
 (17/32) old-nvmetest.py:NVMeTest.testwritezeroes;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: STARTED
 (17/32) old-nvmetest.py:NVMeTest.testwritezeroes;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: PASS (5.65 s)
 (18/32) old-nvmetest.py:NVMeTest.testwritezeroes;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: STARTED
 (18/32) old-nvmetest.py:NVMeTest.testwritezeroes;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: PASS (0.37 s)
 (19/32) old-nvmetest.py:NVMeTest.testwriteuncorrectable;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: STARTED
 (19/32) old-nvmetest.py:NVMeTest.testwriteuncorrectable;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: PASS (5.73 s)
 (20/32) old-nvmetest.py:NVMeTest.testwriteuncorrectable;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: STARTED
 (20/32) old-nvmetest.py:NVMeTest.testwriteuncorrectable;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: PASS (0.37 s)
 (21/32) old-nvmetest.py:NVMeTest.testdsm;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: STARTED
 (21/32) old-nvmetest.py:NVMeTest.testdsm;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: PASS (5.64 s)
 (22/32) old-nvmetest.py:NVMeTest.testdsm;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: STARTED
 (22/32) old-nvmetest.py:NVMeTest.testdsm;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: PASS (0.38 s)
 (23/32) old-nvmetest.py:NVMeTest.testreset;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: STARTED
 (23/32) old-nvmetest.py:NVMeTest.testreset;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: PASS (5.87 s)
 (24/32) old-nvmetest.py:NVMeTest.testreset;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: STARTED
 (24/32) old-nvmetest.py:NVMeTest.testreset;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: PASS (0.65 s)
 (25/32) old-nvmetest.py:NVMeTest.testreset_sysfs;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: STARTED
 (25/32) old-nvmetest.py:NVMeTest.testreset_sysfs;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: PASS (5.84 s)
 (26/32) old-nvmetest.py:NVMeTest.testreset_sysfs;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: STARTED
 (26/32) old-nvmetest.py:NVMeTest.testreset_sysfs;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: PASS (0.59 s)
 (27/32) old-nvmetest.py:NVMeTest.testsubsystemreset;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: STARTED
 (27/32) old-nvmetest.py:NVMeTest.testsubsystemreset;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: CANCEL: NVM Subsystem Reset is not supported (5.69 s)
 (28/32) old-nvmetest.py:NVMeTest.testsubsystemreset;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: STARTED
 (28/32) old-nvmetest.py:NVMeTest.testsubsystemreset;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: CANCEL: NVM Subsystem Reset is not supported (0.36 s)
 (29/32) old-nvmetest.py:NVMeTest.test_create_namespaces;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: STARTED
 (29/32) old-nvmetest.py:NVMeTest.test_create_namespaces;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: PASS (14.23 s)
 (30/32) old-nvmetest.py:NVMeTest.test_create_namespaces;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: STARTED
 (30/32) old-nvmetest.py:NVMeTest.test_create_namespaces;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: PASS (8.80 s)
 (31/32) old-nvmetest.py:NVMeTest.test_delete_all_ns;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: STARTED
 (31/32) old-nvmetest.py:NVMeTest.test_delete_all_ns;run-Devices-device1-firmware_url-namespace_count-package-upstream-nvme-cli-9bc3: PASS (14.08 s)
 (32/32) old-nvmetest.py:NVMeTest.test_delete_all_ns;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: STARTED
 (32/32) old-nvmetest.py:NVMeTest.test_delete_all_ns;run-Devices-device1-firmware_url-namespace_count-package-distro-nvme-cli-5f61: PASS (0.36 s)
RESULTS    : PASS 28 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 4
JOB HTML   : /home/OpTest/avocado-fvt-wrapper/results/job-2023-09-22T00.38-c8bfdfd/results.html
JOB TIME   : 329.01 s
[root@sys ssd]# 
`

**Review by abdhaleegit — CHANGES_REQUESTED** [review]

File: io/disk/ssd/nvmetest.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 from avocado.utils import disk
 from avocado.utils import process
 from avocado.utils import archive
-from avocado.utils import build
+# from avocado.utils import build
```
[the remark this answers is not in the document]
Comment (Naresh-ibm) [reply io/disk/ssd/nvmetest.py:28]: This is not used any where. earlier it was used.
just commented thinking if needed in future.

[the remark this answers is not in the document]
Comment (abdhaleegit) [reply io/disk/ssd/nvmetest.py:28]: remove it

**Review by abdhaleegit — APPROVED** [review]
LGTM

**Pull request merged**